### PR TITLE
Add Vocabulary icon font to the repository

### PR DESCRIPTION
**Pull request by dhruvkb**
**Description**
Downloaded the SVG assets from [Font Awesome](https://github.com/FortAwesome/Font-Awesome/releases) and then used [IconMoon](https://icomoon.io/) to generate an icon font. The icon font only includes the icons mentioned in Figma. The icon font combines icons from 'solid', 'regular' and 'brand' icon sets of Font Awesome.

**Type of PR** 
This PR is a feature.

**Technicalities**
The PR also comes with the CSS required to setup the fonts and use the icons in the form of `<i>` tags.

**Checklist:**
 
- [x] My pull request has a descriptive title (not a vague title like "Update `index.md`").
- [x] My pull request targets the `master` branch of the repository.
- [x] My commit messages follow [best practices][best_practices].
- [x] My code follows the established code style of the repository.
- [ ] I added tests for the changes I made (if applicable).
- [x] I added or updated documentation (if applicable).
- [x] I tried running the project locally and verified that there are no visible errors.

[best_practices]:https://gist.github.com/robertpainsi/b632364184e70900af4ab688decf6f53

**Legal**
By submitting this PR, I agree to abide by the terms of the Developer 
Certificate of Origin.

<details>
<summary>Developer Certificate of Origin</summary>

Developer Certificate of Origin<br>
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.<br>
1 Letterman Drive<br>
Suite D4700<br>
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.

Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
</details>

### CODE REVIEW

**Review by hugosolar — APPROVED** [review]
Using Icomoon was my first approach to this. that's why I think in something more maintainable because in this case if we need to add more icons we should go through icomoon (select the icons again because you can't save the session) and generate the typography again. I mean, it works ok, but it can be improved and automated by using the package to generate the webfont from a folder of SVGs (which can be the same from FontAwesome).
I would say we can leave it this way and discuss later about improving this

**Comment by dhruvkb** [comment]
IcoMoon was more preferable to me instead of using a code based approach as it would allow non-developers to be able to work on the package. I presumed that IcoMoon would be easier to use for someone like @panchovm.

**Comment by fcoveram** [comment]
I really like the idea. Thanks @hugosolar and @dhruvkb

**Comment by dhruvkb** [comment]
In fact, if we can get SVGs in uniform size, we can even make an icon font with the different logos.

File: src/stories/vocabulary_icons.stories.js [hunk 1]
```diff
@@ -0,0 +1,58 @@
+export default {
+  title: 'Vocabulary Icons',
+  decorators: [
+    storyFn => `<div class="has-text-black" style="font-size: 2em;">${storyFn()}</div>`
+  ]
+}
+
+export const Chevrons = () => `
```
Comment (brenoferreira) [inline src/stories/vocabulary_icons.stories.js:8]: Can you rename these icons to `arrows`? `chevrons`  are less obvious and have connotation with the oil company and military symbols

**Review by brenoferreira — CHANGES_REQUESTED** [review]
Just one minor change request, otherwise looks good

File: src/stories/vocabulary_icons.stories.js (hunk 1 above)
Comment (hugosolar) [reply src/stories/vocabulary_icons.stories.js:8]: but those symbols are called chevrons in most of the webfonts (even font awesome), I think we should keep it with that name

Comment (kgodey) [reply src/stories/vocabulary_icons.stories.js:8]: I too think "chevron" is clearer. Arrows are more generic.

Comment (dhruvkb) [reply src/stories/vocabulary_icons.stories.js:8]: There are a lot of [types of pointing shapes](https://fontawesome.com/icons?d=gallery&c=arrows&m=free). I simply used `chevron` as that is what FA called them.

**Review by brenoferreira — APPROVED** [review]

**Pull request merged**